testpilot/frontend/static-src/app/views/experiment-page.js

```diff
@@ -217,6 +248,11 @@ export default PageView.extend({
     return this;
   },
 
+  beforeRender() {
+    this.activeUser = app.me.hasAddon;
+    this.isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
```

Comment:
@chuckharmston To ensure `isFirefox` only flags desktop FF, have a look at [this MDN page](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox).

I'm not totally sure, but I think `Gecko/20100101` is unique to desktop foxes.